tests/cli/functional/actions/test_config_actions.py

```diff
@@ -130,16 +135,19 @@ def test_handle_update_invalid_configuration_file_cli_action(config,
     configure.handle_missing_configuration_file.assert_not_called()
     config._write_configuration_file.assert_not_called()
     configure.handle_invalid_configuration_file.assert_called()
+    captured = capsys.readouterr()
+    assert "Invalid Configuration" in captured.out
 
 
-def test_destroy_configuration_cli_action(config, test_emitter, stdout_trap, mocker, mock_click_confirm):
+def test_destroy_configuration_cli_action(config, test_emitter, capsys, mocker, mock_stdin):
```

Comment:
Do you think it makes sense to still keep these two fixtures decoupled / separate?

Reply:
You mean, `capsys` and `mock_stdin`? It seems to me that `capsys` would be used much more often, sometimes just to suppress the console output. Plus it's not even our own fixture. I'm leaning towards keeping them separate.

Reply:
Perhaps a fixture that's just a wrapper for both - then you just pass a single fixture to the test, and you can either return the internal fixtures themselves or wrap their functionality in pass-through methods. Some sort of `test_io` fixture...?

It could be nice to have test io functionality consolidated in one place.

Reply:
Just returning a tuple of two fixtures would actually make you type more for no benefit. Compare:
```
def test_something(mock_stdin, capsys, ...):
```
vs
```
def test_something(mock_io):
    mock_stdin, capsys = mock_io
```

As for making a wraparound class... Somehow the idea does not appeal to me either. It won't be used to hide any implementations, since all the methods will be essentially pass-through. It will just add a layer of abstraction for no benefit of hiding complexity. Can we have a tiebreaker vote? :)

Reply:
I vote Nay.  Let's keep them separate.

Reply:
I want a recount 😂 - just kidding - I like the 2nd option `mock_stdin, capsys = mock_io` so we pass less parameters - but I know when I'm outnumbered.